tests/functional/object/mcg/test_noobaa_metrics.py

```
@@ -273,3 +273,253 @@ def test_noobaa_space_available_using_cli(
+                bucket_name=bucket_name,
+                key="QuotaStatus",
+            ):
+                last_seen = quota_status
+                if quota_status == QuotaStatus.APPROACHING:
+                    logger.info("Quota status is APPROACHING_QUOTA as expected")
+                    break
+        except TimeoutExpiredError:
+            assert False, (
+                f"Quota status did not reach {QuotaStatus.APPROACHING} "
+                f"within 360s, last observed: {last_seen}"
+            )
```

Ruff (0.15.21)

[warning] Assertion always fails, replace with `pytest.fail()`

(PT015)